pkg/cmd/infra/router/f5.go

```diff
@@ -225,7 +225,8 @@ func (o *F5RouterOptions) Run() error {
 
 	factory := o.RouterSelection.NewFactory(oc, kc)
 	watchNodes := (len(o.InternalAddress) != 0 && len(o.VxlanGateway) != 0)
-	controller := factory.Create(plugin, watchNodes)
+	// TODO should ingress be configurable for f5?
```

Comment:
Can the existing plugin support it?  We already set the --enable-ingress on the router, may as well use it here if the router is configured for f5.